code/modules/clothing/spacesuits/void/station.dm

```diff
@@ -125,6 +125,46 @@
 /obj/item/clothing/suit/space/void/medical/equipped
 	boots = /obj/item/clothing/shoes/magboots
 
+/obj/item/clothing/head/space/void/paramedic
+	name = "Moebius Paramedic's Voidsuit Helmet"
```

Comment:
why capitalized

Reply:
its usually what i do, but i will close this pr and reopen a new one with proper changes